### DISCUSSION TITLE
Can't connect to cyberduck

### DISCUSSION THREAD

**Discussion by Ratcliffegds** [iterate-ch/cyberduck#14751] [Q&A]
[
![Cyberduck Error](https://github.com/iterate-ch/cyberduck/assets/29524184/af61c49b-13d9-471d-b394-e5cad40fa824)
](url)

**Comment by tatze96** [comment]
Please attach a screenshot of the connection error.

Additionally, can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), try to connect to the server, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14751 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
